lib/moesif_rack/moesif_middleware.rb

```diff
@@ -100,6 +101,26 @@ def @moesif_helpers.log_debug(message)
       end
     end
 
+    def parse_multipart(multipart_form_data, content_type)
+      @moesif_helpers.log_debug("try to parse multiple part #{content_type}")
+
+      sanitized_multipart_form_data = multipart_form_data.gsub(/\r?\n/, "\r\n")
+
+      io = StringIO.new(sanitized_multipart_form_data)
+      tempfile = Rack::Multipart::Parser::TEMPFILE_FACTORY
+      bufsize = Rack::Multipart::Parser::BUFSIZE
+      query_parser = Rack::Utils.default_query_parser
+      result = Rack::Multipart::Parser.parse(io, sanitized_multipart_form_data.length, content_type, tempfile, bufsize, query_parser)
+
+
+      @moesif_helpers.log_debug("result before selection")
```

Comment:
nitpic: We'd print this only if debug flag is set to true to reduce noise

Reply:
ok. I think the print is a lot here. let me remove some of the stuff.